lib/public/components/tag/tagDetail.js

```diff
@@ -115,21 +116,10 @@ const activeFields = (detailsModel) => ({
         visible: true,
         classes: 'w-10 f6',
         format: (color) => detailsModel.isEditModeEnabled ?
-            h('input#color.form-control.w-15.mb2', {
-                placeholder: 'Enter the color#...',
-                value: detailsModel.tagChanges['color'],
-                minlength: 6,
-                maxlength: 6,
-                oninput: (e) => {
-                    if (e.target.value !== '') {
-                        detailsModel.setTagChange('color', e.target.value);
-                    } else {
-                        detailsModel.setTagChange('color', null);
-                    }
-                },
-                style: { backgroundColor: `#${detailsModel.tagChanges['color']}` },
+            colorInputComponent(detailsModel.tagChanges['color'], (e) => {
+                detailsModel.setTagChange('color', (e.target.value || null).replace(/^#/, ''));
             }) :
-            h('.rounded-1', { style: { backgroundColor: `#${color}` } }, color ? `#${color}` : ''),
+            h('.br2', { style: { backgroundColor: `#${color}` } }, color ? `#${color}` : ''),
```

Comment:
If you have no color, backgroundColor will be '#' which is invalid, you should include the style internary too.